czech/datahandler_user.lang.php

```diff
@@ -42,8 +42,8 @@
 $l['userdata_bad_profile_field_values'] = 'Nezvolil(a) jste platnou možnost pro pole "{1}". Prosím, vyberte ji z uvedených možností.';
 $l['userdata_max_limit_reached'] = 'Zadal(a) jste nesprávný počet znaků pro pole "{1}". Prosím, nezadávejte do tohoto pole více než {2} znaků.';
```

Comment:
vykání?

Reply:
Na nějakém německém fóru, které také běželo na MyBB jsem viděl možnost, která se mi líbila a to jest, že mají jazykový balíček pro týkání a vykání nicméně bude to dost náročné na udržování a proto zatím je ve výchozím stavu vykání.